command/snapshot/inspect/snapshot_inspect.go

```diff
@@ -196,12 +242,49 @@ func enhance(file io.Reader) (map[structs.MessageType]typeStats, int, error) {
 		s.Count++
 		totalSize = cr.read
 		stats[msg] = s
+
+		if detailed {
+			if s.Name == "KVS" {
```

Comment:
We could remove the case and have something like the following: 
```
valz := val.(map[string]interface{})
	for k, v := range valz {
		if k != "Key" {
			continue
		}
```
^ Notice the if not check